worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -957,22 +956,13 @@ class PushDataHandler extends BaseMessageHandler with Logging {
       callback: RpcResponseCallback,
       wrappedCallback: RpcResponseCallback): Boolean = {
```

Comment:
wrappedCallback is not used. BTW, ```locationIsNull```, ```handleRpcRequestCore``` are only for MapPartition, I think better to refactor in the future.

Reply:
> wrappedCallback is not used. BTW, `locationIsNull`, `handleRpcRequestCore` are only for MapPartition, I think better to refactor in the future.

Agree.